vcpkg/github-integration.md

```diff
@@ -0,0 +1,94 @@
+
+## The GitHub dependency graph
+
+[!INCLUDE [experimental](../includes/experimental.md)]
+
+> [!TIP]
+> We are actively developing this feature and would like to hear your feedback. Let us know if you have any thoughts about the current functionality or any desired functionality by emailing [vcpkg@microsoft.com](mailto:vcpkg@microsoft.com) or by filing an issue in [vcpkg-docs](https://github.com/microsoft/vcpkg-docs/issues). If you have feedback on any of the GitHub features that this enables, let us know that too and we'll make sure it gets to the right folks at GitHub.
+
+### About the GitHub dependency graph
+
+The GitHub [dependency graph](https://docs.github.com/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph) stores the set of dependencies for a repository. Beyond being able to visualize a repository's dependencies, GitHub builds several useful features on top of this data, including [dependency review](https://docs.github.com/code-security/supply-chain-security/understanding-your-software-supply-chain/about-dependency-review) and [Dependabot alerts](https://docs.github.com/code-security/dependabot/dependabot-alerts/about-dependabot-alerts).
+
+### vcpkg integration with the GitHub dependency graph
+
+vcpkg has experimental support for populating the GitHub dependency graph from within a GitHub Actions workflow. To enable this feature, make the following changes in your workflow file:
+
+* Set the `VCPKG_FEATURE_FLAGS` environment variable to include the value `dependencygraph`. 
+* Set the `GITHUB_TOKEN` environment variable to the value `${{ secrets.GITHUB_TOKEN }}`.
+* Give GitHub Actions write access to the contents of your repository by including the following permissions block. This is required to write the dependency graph metadata to your repository. This __will not__ add any commits to your repository or make any other modification to the contents of your repository.
+
+```yaml
+permissions:
+  contents: write
+```
+
+You must enable the GitHub dependency graph in your repository's settings (enabled by default on public repositories). For private repositories, follow the [GitHub documentation to enable the dependency graph](https://docs.github.com/code-security/supply-chain-security/understanding-your-software-supply-chain/configuring-the-dependency-graph#enabling-and-disabling-the-dependency-graph-for-a-private-repository) in order to satisfy this requirement.
+
+### Known limitations
+
+The following are known limitations in the current implementation. __Even though we know of these, if they affect the value for you, we'd still like to hear from you so that we can prioritize what we need to fix and how we should fix it.__
```

Comment:
What's with the double underscores? If something is important, use the standard > [!IMPORTANT] or > [!NOTE]   Bold runs afoul of our typical formatting for other things.

Reply:
Removing bolding in next update. It's not important enough to call out that prominently with `> [!IMPORTANT]` or `> [!NOTE]`.